lib/queuePopulator/LogReader.js

```diff
@@ -522,6 +484,94 @@ class LogReader {
         return undefined;
     }
 
+    _processLogEntry(batchState, record, entry, cb) {
+        function _executeFilter(ext, entry, cb) {
+            if (typeof ext.filterAsync === 'function') {
+                ext.filterAsync(entry, cb);
+            } else {
+                ext.filter(entry);
+                process.nextTick(cb);
+            }
+        }
+        async.eachSeries(this._extensions, (ext, next) => {
+            /**
+             * Delete entries don't have a value field.
+             * Value field normally contains the timestamp
+             * of the event, which is needed for the notification
+             * extension.
+             * That gets added here, we use the oplog event timestamp
+             */
+            if (
+                entry.type === 'delete' &&
+                ext.constructor.name === 'NotificationQueuePopulator'
+            ) {
+                entry.value = JSON.stringify({
```

Comment:
Same comment here for the JSON.stringify

Reply:
No need i think inside are values we set

https://github.com/scality/Arsenal/blob/c343820cae2c7a533fb73046b2ad4c13cb77fc8b/lib/storage/metadata/mongoclient/ListRecordStream.js#L133-L137